src/DashboardService.Models.Dto/Requests/Group/Filters/GetGroupFilter.cs

```diff
@@ -0,0 +1,13 @@
+﻿using LT.DigitalOffice.Kernel.Requests;
+using Microsoft.AspNetCore.Mvc;
+using System;
+
+namespace LT.DigitalOffice.DashboardService.Models.Dto.Requests.Group.Filters;
+public record GetGroupFilter : BaseFindFilter
+{
+  [FromQuery]
+  public Guid? BoardId { get; set; }
```

Comment:
you dont need BoardId here - you already get it from route

Reply:
++